app/views/content/jobseeker-guides/3-quick-ways-to-find-the-right-teaching-job.md

```diff
@@ -0,0 +1,32 @@
+---
+order: 500
+title: 3 quick ways to find the right teaching job for you
+meta_description: Congratulations, you’ve been invited to an interview for a teaching job! What’s next? We’ve spoken to experienced teachers to get their top tips for success including how to prepare answers to interview questions.
+date_posted: 07/05/2024
```

Comment:
@alexleedfe assuming this is just a classic gov.uk/GDS pattern, but wondering if we need/how helpful the date posted is?